packages/kbn-palettes/palettes/index.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { KbnPalettes } from '../classes/palettes';
+import { elasticPalette, getNeutralPalette } from './categorical';
+import { elasticClassicPalette, kibana4Palette, kibana7Palette } from './legacy/categorical';
+import {
+  complementaryPalette,
+  coolPalette,
+  grayPalette,
+  greenPalette,
+  redPalette,
+  statusPalette,
+  temperaturePalette,
+  warmPalette,
+} from './gradient';
+export { logLevelPalette } from './semantic';
+
+export const getPalettes = (darkMode: boolean) =>
+  new KbnPalettes(
+    [
+      elasticPalette,
+      kibana7Palette,
+      kibana4Palette,
+      getNeutralPalette(darkMode),
+      complementaryPalette,
+      coolPalette,
+      grayPalette,
+      greenPalette,
+      redPalette,
+      statusPalette,
+      temperaturePalette,
+      warmPalette,
+      elasticClassicPalette,
+    ],
+    elasticPalette
+  );
```

Comment:
This brings up a good point...

Originally I was doing this because the eui palette functions need to be reinvoked when the theme changes. Since I don't have a direct import like `getEuiCoolPalette({ name: 'Borealis', darkMode: false })` I cannot enumerate all possible static palette combinations.

However, looking at this code again it never reinvokes the eui palette methods. Take the `elasticPalette` for example, this only calls `euiPaletteColorBlind` once when the object in created. This is likely why I had issues with the late update of eui colors.

But since eui fixed the vis color store to immediately instantiate based on the current theme without a delayed update, we no longer need to reinvoke the palette functions, at least not until we need to make them reactive in #203710.

I created static exports in 978251de6dc366043f584d4cf73a073c5c737ff4 and still looks to work correctly.